asb-ballerina/tests/asb_negative_test.bal

```diff
@@ -33,10 +33,12 @@ function testReceivePayloadWithIncorrectExpectedType() returns error? {
     MessageReceiver messageReceiver = check new (receiverConfig);
     log:printInfo("Receiving from Asb receiver client.");
 
-    float|error? expectedPayload = messageReceiver->receivePayload(serverWaitTime);
+    float|Error? expectedPayload = messageReceiver->receivePayload(serverWaitTime);
     log:printInfo("Asserting received payloads.");
-    test:assertTrue(expectedPayload is error, msg = "Unexpected payload received");
-    test:assertEquals((<error>expectedPayload).message(), "{ballerina/lang.value}ConversionError");
+    test:assertTrue(expectedPayload is Error, msg = "Unexpected payload received");
+    test:assertEquals((<Error>expectedPayload).message(), "Failed to deserialize the message payload " +
```

Comment:
Can these messages change in future lang versions? :) Thinking if we should match the exact.

Reply:
error message in L39 is a error message we provide from the connector. So it won't change with the lang version :)